lib/resources/r4/observation.yaml

```diff
@@ -92,14 +92,12 @@ fields:
       "code": {
         "coding": [
           {
-            "system": 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/72',
-            "code": '4154126',
-            "display": 'Height/Length Measured',
-            "userSelected": true
+            "system": 'http://loinc.org',
+            "code": '8331-1'
           }
         ],
-        "text": 'Height/Length Measured'
-      }
+        "text": 'Temperature Oral'
+      },
```

Comment:
i think this is an extra coma?

Reply:
fix for this 
a0eb984[](https://github.com/cerner/fhir.cerner.com/pull/472/commits/a0eb984f468d92092fd31cd80ac6a6ddedd08bf6)

Reply:
<img width="1162" alt="obs_3" src="https://user-images.githubusercontent.com/71471042/94178695-694e9400-fe69-11ea-9f48-c5bd1354ec4a.png">